sofiadinizms/IP-Project
# Feat/background screen maps

## What I did

- Added Assets and created Backgrounds
- Created Menu and Game over screen
- Made maps appear randomly on the game grid
  - Detailing

## How to test

1. Run the project
2. Press any key to play
3. Play
4. Return to home screen after you lose
5. Check if maps appear randomly while you play
